src/ert/gui/tools/workflows/run_workflow_widget.py

```diff
@@ -21,12 +27,13 @@ class RunWorkflowWidget(QWidget):
     workflowFailed = Signal()
     workflowKilled = Signal()
 
-    def __init__(self, ert):
+    def __init__(self, ert, storage, notifier):
```

Comment:
Types for storage and notifier? Perhaps `notifier `is enough or is the `notifier.storage` a different one ?